docs/antrea-ipam.md

```diff
@@ -218,6 +218,34 @@ metadata:
 A StatefulSet Pod's IP will be kept after Pod restarts, when the IP is allocated from the
 annotated IPPool.
 
+### IPAM behaviors
+
+When `AntreaIPAM` is enabled, `antrea-controller` will watch IPPool CRs and StatefulSets
+from K8s apiserver, and `antrea-agent` will register IPAM driver to CNI server.
+
+#### On IPPool CR create/update event
+
+`antrea-controller` will update IPPool counters, and periodically clean up stale IP addresses.
+
+#### On StatefulSet create event
+
+`antrea-controller` will check annotations and preallocate IPs from the IPPool.
```

Comment:
How about StatefulSet update event? what will happen if the user remove the annotations 'ipam.antrea.io/ippools' of a StatefulSet?

Reply:
Btw, please help to squash the commits.

Reply:
No need to handle StatefulSet update event.
Change ippool annotations is not supported as explained in this doc:
```
Note that the IP pool annotation cannot be updated or deleted without recreating
the resource.
```

Reply:
Squashed.